conans/test/integration/command_v2/list_recipe_revisions_test.py

```diff
@@ -27,15 +31,15 @@ def _upload_recipe(self, remote, reference):
 
 
 class TestParams(TestListRecipeRevisionsBase):
-    def test_fail_if_reference_is_not_correct(self):
-        self.client.run("list recipe-revisions whatever", assert_error=True)
-        assert "ERROR: Specify the 'name' and the 'version'" in self.client.out
-
-        self.client.run("list recipe-revisions whatever/", assert_error=True)
-        assert "ERROR: Specify the 'name' and the 'version'" in self.client.out
-
-        self.client.run("list recipe-revisions whatever/1", assert_error=True)
-        assert "ERROR: Value provided for package version, '1' (type Version), is too short" in self.client.out
+    @pytest.mark.parametrize("ref", [
```

Comment:
This test is repeated several times, maybe extract to a ``list_ux_test.py`` file?

Reply:
This is not the only one. I was thinking of having only one module parameterized with all the list-subcommands.